src/query/plan/profile.hpp

```diff
@@ -29,7 +29,7 @@ struct ProfilingStats {
   int64_t actual_hits{0};
   unsigned long long num_cycles{0};
   uint64_t key{0};
-  const char *name{nullptr};
+  std::string name;
```

Comment:
`std::string_view`

Reply:
perhaps

Reply:
Checked, changing it to `std::string_view` would need a cascade of changes in the classes/structs that deal with profiling results.